src/backend/services/ClanManagementService.js

```diff
@@ -36,6 +36,24 @@ class ClanManagementService {
         }
     }
 
+    async transferOwnership(newOwnerMemberId, clanId) {
+        const newOwnerUser =
+            await this.clanService.getClanMembership(newOwnerMemberId)
```

Comment:
I'm unsure that this returns what I want it to:

need to patch in the new owner like:
```
leader: {
  data: {
      id: newOwnerId,
      type: 'player',
```
So i need the player ID retrieved from the membershipID. Not seeing any documentation for this but its based on how this was previously implemented + a bit of faith.

@fcaps if you can shed some light on this then it should be good for testing before merge

Reply:
i had to look into it and thought i could directly fix it.
in short: the clan.members already got a playId, so we don't need the memebrship.
https://github.com/beckpaul/faf-website/pull/2